python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -74,8 +79,20 @@ def __init__(self, config: Mapping[str, str], log):  # pylint: disable=too-many-
         self.autocommit = self.conn_args.get("autocommit", False)
         self.log = log
 
+    @public
```

Comment:
@smackesey I'm still getting a bit used to what should/should not get teh `@public` decorator. This was the only way I got these functions to appear in the API docs, but want to confirm that it's ok to be marking these public

Reply:
These are the rules that govern our guarantees for public APIs: https://docs.dagster.io/getting-started/releases